pymongo/mongo_client.py

```diff
@@ -1487,14 +1487,35 @@ def __getitem__(self, name):
         """
         return database.Database(self, name)
 
-    def _close_cursor(self, cursor_id, address):
-        """Send a kill cursors message with the given id.
+    def _cleanup_cursor(self, locks_allowed, cursor_id, address, sock_mgr,
+                        session, explicit_session):
+        """Cleanup a cursor."""
```

Comment:
Can you update this docstring to be more descriptive? Its not clear how exactly this method is different from the other `_close_cursor*` methods from this docstring... Some more detail would be useful.

Reply:
Done.